VideoLocker/src/main/java/org/edx/mobile/user/UserAPI.java

```diff
@@ -68,7 +86,78 @@ public void deleteProfileImage(@NonNull String username) throws RetroHttpExcepti
         return userService.getBadges(username, page, ApiConstants.STANDARD_PAGE_SIZE);
     }
 
-    public List<EnrolledCoursesResponse> getUserEnrolledCourses(@NonNull String username) throws RetroHttpException {
-        return userService.getUserEnrolledCourses(username);
+    public List<EnrolledCoursesResponse> getUserEnrolledCourses(@NonNull String username, boolean tryCache) throws RetroHttpException {
+        List<EnrolledCoursesResponse> ret = null;
+        String json = null;
+
+        final String cacheKey = config.getApiHostURL() + "/api/mobile/v0.5/users/" + username + "/course_enrollments";
```

Comment:
It's essential that this key be the same as the one in Api.java.

Could we have share the same method/declaration to ensure that?